dependencies/bionic/kafo/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-kafo (3.0.0-1) stable; urgency=low
```

Comment:
Was this generated from a script? I guess I'm not sure what the urgency part indicates here, or is it just to say "Hey this isn't needed RIGHT NOW"?

Reply:
debian changelogs contain an `urgency` tag, to indicate how quickly packages should flow from `unstable` to `testing`. `low` means 10 days delay, `medium` 5 and `high` 2. as we don't upload to Debian proper, this field is irrelevant for us (but it needs to be set for the changelog entry to be valid)

Reply:
And to answer the generated part: https://github.com/theforeman/foreman-packaging/blob/deb/develop/scripts/changelog.rb

Reply:
Thanks for the knowledge!